app/src/main/java/org/jellyfin/androidtv/preference/IUserPreferences.kt

```diff
@@ -0,0 +1,4 @@
+package org.jellyfin.androidtv.preference
+
+// Stub interface to allow Koin to resolve between different preference stores
+interface IUserPreferences : PreferenceStore
```

Comment:
I don't see a point in an empty interface either, the way preference stores are meant to work is to have a class extending one of the preference store implementations (right now that is sharedprefs or displayprefs). No interfaces needed or desired.